dist/css/bootstrap.css

```diff
@@ -1442,6 +1442,26 @@ h6,
 .h6 {
   color: #829ca9;
 }
+.display-1 {
+  font-size: 6rem;
```

Comment:
All our line-heights should be a multiple of 8px, although font-size can be anything.